exporter/metric/metric.go

```diff
@@ -223,29 +223,48 @@ func (me *metricExporter) createMetricDescriptorIfNeeded(ctx context.Context, md
 func (me *metricExporter) exportTimeSeries(ctx context.Context, rm metricdata.ResourceMetrics) error {
 	tss := []*monitoringpb.TimeSeries{}
 	mr := me.resourceToMonitoredResourcepb(rm.Resource)
-	var aggError error
 
+	var aggErrors []error
 	extraLabels := me.extraLabelsFromResource(rm.Resource)
 	for _, scope := range rm.ScopeMetrics {
 		for _, metrics := range scope.Metrics {
 			ts, err := me.recordToTspb(metrics, mr, scope.Scope, extraLabels)
-			aggError = multierr.Append(aggError, err)
+			if err != nil {
+				aggErrors = append(aggErrors, err)
+			}
+
 			tss = append(tss, ts...)
 		}
 	}
 
 	if len(tss) == 0 {
-		return aggError
+		return multierr.Combine(aggErrors...)
 	}
 
-	// TODO: When this exporter is rewritten, support writing to multiple
-	// projects based on the "gcp.project.id" resource.
-	req := &monitoringpb.CreateTimeSeriesRequest{
-		Name:       fmt.Sprintf("projects/%s", me.o.projectID),
-		TimeSeries: tss,
+	name := fmt.Sprintf("projects/%s", me.o.projectID)
+
+	const batchSize = 200
```

Comment:
could you define this at the top-level of the file with a comment explaining it, like in the collector exporter https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/blob/e9bb597e4aad734e5fa68ac8847e7000303368de/exporter/collector/metrics.go#L104-L108